tsc/OpenTimelineIO_TSC_Charter.md

```diff
@@ -0,0 +1,67 @@
+  * **f.** The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC. The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and the Technical Advisory Council of the Academy Software Foundation of The Linux Foundation.
+  * **g.** Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
+    * **i.** coordinating the technical direction of the Project;
+    * **ii.** approving project or system proposals (including, but not limited to, incubation, deprecation, and changes to a sub-project's scope);
+    * **iii.** organizing sub-projects and removing projects;
+    * **iv.** creating sub-committees or working groups to focus on cross-project technical issues and requirements;
+    * **v.** appointing representatives to work with other open source or open standards communities;
+    * **vi.** establishing community norms, workflows, issuing releases, and security issue reporting policies;
+    * **vii.** approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the Series Manager to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
+    * **viii.** discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+    * **ix.** coordinating any marketing, events, or communications regarding the Project with the LF Projects Manager or their designee.
+
+## 3. TSC Voting
+  * **a.** While the Project aims to operate as a consensus based community, if any TSC decision requires a vote to move the Project forward, the voting members of the TSC will vote on a one vote per voting member basis.
+  * **b.** Quorum for TSC meetings requires at least two-thirds of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met, but will be prevented from making any decisions at the meeting.
+  * **c.** Except as provided in Section 7.c. and 8.a, decisions by vote at a meeting require a majority vote of those in attendance, provided quorum is met. Decisions made by electronic vote without a meeting require a majority vote of all voting members of the TSC.
+  * **d.** In the event a vote cannot be resolved by the TSC, any voting member of the TSC may refer the matter to the Series Manager or its designee for assistance in reaching a resolution.
+
+## 4. Compliance with Policies
+  * **a.** This Charter is subject to the Series Agreement for the Project and the Operating Agreement of LF Projects. Contributors will comply with the policies of LF Projects as may be adopted and amended by LF Projects, including, without limitation the policies listed at https://lfprojects.org/policies/.
```

Comment:
The originating boilerplate is from the LF. I wonder if we were meant to /s/LF/ASWF/ in section 4?